assets/js/src/bulk-delete.js

```diff
@@ -204,6 +204,14 @@ jQuery(document).ready(function () {
 			return false;
 		}
 	};
+
+	BulkWP.validateUserRole = function() {
+		if (null !== jQuery('#smbd_u_role_roles').val()) {
```

Comment:
Use an class name instead of id, so that this validation function can be used in all places where we are having a user dropdown.

Also the above 4 lines can be reduced to a single line. Eg: `return (null !== jQuery('#smbd_u_role_roles').val())`

Reply:
@sudar I added 'id' attribute in user dropdown in renderer.php.  So, in all places id  attribute will appear.  

I will modify the code, to make it in a single line.